app/org/maproulette/Config.scala

```diff
@@ -37,6 +37,8 @@ class Config @Inject()(implicit val configuration: Configuration) {
     this.config.getOptional[Boolean](Config.KEY_DEBUG).getOrElse(false)
   lazy val isDevMode: Boolean =
     this.config.getOptional[Boolean](Config.KEY_DEVMODE).getOrElse(false)
+  lazy val mockOSM: Boolean =
```

Comment:
I think you should change the variable name here to reference the fact that you are using this to mock changeset submission to OSM. Otherwise it might be confusing about what this actually does.